scripts/add-comment-to-customer.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE customer ADD COLUMN comment text;
```

Comment:
Do we need to set the default value to something or make it nullable?

Reply:
seems not: 
![image](https://user-images.githubusercontent.com/6697636/66469983-1c6f8300-ea89-11e9-8640-941eb0d44f69.png)